### PR TITLE
Add link to psalm.el and replace PHPStan's new web site

### PR DESCRIPTION
**Pull request by zonuexe**
I created [psalm.el](https://github.com/emacs-php/psalm.el) as Flycheck extension.

refs https://github.com/flycheck/flycheck/pull/1765 and https://github.com/melpa/melpa/pull/6897

### CODE REVIEW

File: doc/community/extensions.rst [hunk 1]
```diff
@@ -134,8 +134,10 @@ PHP
 ---
 
 * :gh:`emacs-php/phpstan.el` adds a PHP static analyzer using PHPStan_.
+* :gh:`emacs-php/psalm.el` adds a PHP static analyzer using PSalm_.
```
Comment (cpitclaudel) [inline doc/community/extensions.rst:137]: Is that the right capitalization?

**Comment by cpitclaudel** [comment]
Thanks a lot! :) I added just one small comment regarding the capitalization of Psalm.

**Comment by zonuexe** [comment]
@cpitclaudel Thank you. I fixed and force-pushed.

**Comment by cpitclaudel** [comment]
:+1:

**Pull request merged**